### PR TITLE
Curvedrive fixes, updates, and comments

### PR DESCRIPTION
**Pull request by Vanilla-Rain**
Makes curvedrive interpolate using equally spaced tangents instead of following the curve directly. Does need some tuning and fixing but can currently work semi-accurately and very consistently at high speeds. GreyingJay's commits here are actually mine but on his computer.

### CODE REVIEW

File: src/org/usfirst/frc/team2706/robot/Robot.java [hunk 1]
```diff
@@ -80,9 +81,11 @@ public void robotInit() {
                         /* no switch: do nothing */ new ArcadeDriveWithJoystick(),
                        /* position 1: do nothing */ new ArcadeDriveWithJoystick(),
             /* position 2: Move Forward one foot */ new StraightDriveWithEncoders(0.3, 2, 1, 5, "AutoForwardFoot"),
-                                                    new RotateDriveWithGyro(0.5, 90, 5, "AutoTurn90"),
+                                                    new RotateDriveWithGyro(0.5, 90, 5, "AutoTurn900"),
```
Comment (eandr127) [inline src/org/usfirst/frc/team2706/robot/Robot.java]: This should be reverted.

File: src/org/usfirst/frc/team2706/robot/Robot.java [hunk 2]
```diff
@@ -123,7 +126,7 @@ public void autonomousInit() {
         Log.i("Robot", "Entering autonomous mode");
         
         driveTrain.reset();
-
+
```
Comment (eandr127) [inline src/org/usfirst/frc/team2706/robot/Robot.java]: Unnecessary

File: src/org/usfirst/frc/team2706/robot/commands/autonomous/experimential/curvedrive/CubicEquation.java [hunk 3]
```diff
@@ -2,28 +2,17 @@
 
 import java.text.DecimalFormat;
 
-/**
```
Comment (eandr127) [inline src/org/usfirst/frc/team2706/robot/commands/autonomous/experimential/curvedrive/CubicEquation.java]: Why is this removed?

File: src/org/usfirst/frc/team2706/robot/Robot.java (hunk 1 above)
Comment (Vanilla-Rain) [reply src/org/usfirst/frc/team2706/robot/Robot.java]: Thats your fault for breaking it with config
